en/releases/main.md

```diff
@@ -1,6 +1,6 @@
 # PX4-Autopilot Main Release Notes
 
-This contains changes to PX4 since the last major release (v1.14).
+This contains changes to PX4 since the last major release (v1.14). 
```

Comment:
@RomanBapst When you're done, can you add a release note in this doc for the change following the pattern in https://docs.px4.io/main/en/releases/1.14.html

I'm trying to make sure that we don't have to come back and think about what we did and why when we get to the release.

Reply:
@RomanBapst ^^^^